blocks/init/src/Blocks/components/paragraph/components/paragraph-editor.js

```diff
@@ -31,15 +39,152 @@ export const ParagraphEditor = (attributes) => {
 		selector(blockClass, blockClass, selectorClass),
 	]);
 
+	/**
+	 * Returns string parsed as nodes, ready for inserting into RichText.
+	 *
+	 * @param {string} inputText HTML string for parsing.
+	 */
+	const processTag = (inputText) => {
```

Comment:
Could you maybe move all of these functions to separate files in the `blocks/init/src/Blocks/components/paragraph/components` folder and then import it from those files in `paragraph-editor`? That way we can keep the `paragraph-editor` more clean.